app.py

```diff
@@ -37,13 +37,13 @@
 @app.route('/', methods=['GET'])
 def index():
     print('Request for index page received')
-    restaurants = Restaurant.query.all()    
+    restaurants = Restaurant.query.all()
     return render_template('index.html', restaurants=restaurants)
 
 @app.route('/<int:id>', methods=['GET'])
 def details(id):
     restaurant = Restaurant.query.where(Restaurant.id == id).first()
-    reviews = Review.query.where(Review.restaurant==id)
+    reviews = Review.query.where(Review.restaurant == id)
```

Comment:
Black adds spaces around operators (notice that line 45 already had that, so this change also increases internal consistency)